tools/windows/DatadogAgentInstaller/WixSetup/Datadog/AgentInstaller.cs

```diff
@@ -223,6 +223,16 @@ public Project ConfigureProject()
                 document
                     .Select("Wix/Product/InstallExecuteSequence")
                     .AddElement("DeleteServices", value: "(Installed AND (REMOVE=\"ALL\") AND NOT (WIX_UPGRADE_DETECTED OR UPGRADINGPRODUCTCODE))");
+                document
```

Comment:
Add comment that explains we're doing it this way because wixsharp `Merge` doesn't work.